packages/core/src/components/radio-pill/_radio-pill.scss

```
@@ -0,0 +1,67 @@
+@import '../../global/variables';
+
+@include exports('ray-radio-pill') {
+  .#{$ray-class-prefix}radio-pill__wrapper {
+    width: 100%;
+    display: flex;
+  }
+
+  .#{$ray-class-prefix}radio-pill {
+    float: left;
```

```suggestion
    float: left;
```

this is not needed when the parent element is flexed

---

fixed